src/store/AppInfo.ts

```diff
@@ -53,7 +53,7 @@ const appInfoState: AppInfoState = {
     hasControlsDisabled: false,
     controlsDisabledMessage: '',
     faucetUrl: undefined,
-    settings: settingService.getProfileSettings(ANON_PROFILE_NAME, NetworkType.TEST_NET), // TODO how to fix here? why static?
+    settings: settingService.getProfileSettings(ANON_PROFILE_NAME), // TODO how to fix here? why static?
```

Comment:
Still static. ;)

Reply:
I did adress this in a different PR https://github.com/symbol/desktop-wallet/pull/1801